docs/contributor_guidelines.rst

```diff
@@ -1,12 +1,27 @@
 Contributing Guidelines
 =======================
 
-Signing commits
+Signing Commits
 ---------------
 
 Commits should be signed, as `explained in the GitHub documentation <https://docs.github.com/en/github/authenticating-to-github/signing-commits>`_.
 This helps verify commits proposed in a pull request are from the expected author.
 
+Code Review
+-----------
+Changes to SecureDrop code or documentation (in any repository) must be reviewed
+by a SecureDrop maintainer.  This is generally done in the form of GitHub pull
+requests (PRs), which are formally reviewed and merged.
+
+If multiple individuals collaborate on a single PR, it is acceptable for
+cross-review by the authors to take place during development, but there must not be
+any unreviewed changes in the pull request at the time of approval.
```

Comment:
This is beyond the scope of what we discussed but I figured would be good to call out explicitly as I think it's existing practice, but happy to tweak.

Reply:
Agreed that this is existing practice.